extensions/random-number/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "random-number",
+  "title": "Random Number",
```

Comment:
Only comment I have is about the name. Since this is a very specific extension, I think having such a generic name is misleading. 

How about "Random US Phone Number"?

Reply:
Hey, I renamed the extension as well as the directory